Statistics/Statistics.cs

```diff
@@ -1,12 +1,32 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace Statistics
 {
     public class StatsComputer
-    {
-        public Stats CalculateStatistics(List<float> numbers) {
+    { 
+        public Stats CalculateStatistics(List<double> numbers)
+        {
             //Implement statistics here
+            IgnoreNaNValues(numbers);
+            Stats computedStats = new Stats();
+            if (numbers.Count == 0)
+            {
+                return computedStats;
+            }
+            computedStats.Average = numbers.Average();
+            computedStats.Min = numbers.Min();
+            computedStats.Max = numbers.Max();
+
+            return computedStats;
+        }
+
+        public List<double> IgnoreNaNValues(List<double> numbers)
+        {
+            numbers.RemoveAll(double.IsNaN);
```

Comment:
Clean removal of NaNs 👍 
However, this will remove the NaNs in the array within the caller too. As a caller of the CalculateStatistics method, I do not expect (seeing the name of the function) that my array will be changed. Maybe I would like to count the number of NaNs (e.g., to see how many invalid inputs were there) after calculating statistics.
1. Write a test case to expose this issue: Compare the size of the array before and after CalculateStatistics and assert them to be the same
2. Resolve the issue and pass this test case.